src/_variables.less

```diff
@@ -125,6 +125,15 @@
 @toggleSwitchOnPosition: translateX(calc(@toggleWidth * 0.4));
 /* END VARIABLES IN special/_toggle.less */
 
+/* BEGIN VARIABLES IN special/_western.less*/
+@choco: #59452b;
```

Comment:
Please rename all your variables to start with western (for example `@westernChoco`) that makes it easier to maintain later on.